workloads/virtual-machines/virtualized-hardware-configuration.md

```diff
@@ -278,3 +278,37 @@ Various VM resources, such as a video adapter, IOThreads, and supplementary syst
 
 Please see [how Pods with resource requests are scheduled](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#how-pods-with-resource-requests-are-scheduled) for additional information on resource requests and limits.
 
+## Hugepages
+
+KubeVirt give you possibility to use hugepages as backing memory for your VM. You will need to provide desired size of hugepages to use, for example for x86_64 architecture it can be `2Mi`.
+
+```yaml
+apiVersion: kubevirt.io/v1alpha1
+kind: VirtualMachine
+metadata:
+  name: myvm
+spec:
+  domain:
+    resources:
+      requests:
+        memory: "64Mi"
+    hugepages:
```

Comment:
Instead of a top-level hugepages item it looks more sane to have it as a parameter to a `memory` struct, in order to give it some smenatics, iow:

```yaml
spec:
  domain:
    resources:
      requests:
        memory: "64Mi"
    memory:
      hugepages:
        size: "2Mi"
```

Reply:
@fabiand I thought about it, but it looks a little bit confusing to have `resources.requests.memory` and `memory.hugepages`
What do you think?

Reply:
We need to look at the VM.

For the VM, regardless if it's plain memory or hugepage, the VM will always just get memory.
That is why it's okay that we only have a memory request field. Because it's about the vm memory, not about what we technically need on the pod to implement this.